### ISSUE TITLE
Error when building Trinity

### ISSUE THREAD

**Issue by saxocellphone** [phaistos-networks/Trinity#15]
Hi, I'm trying to build Trinity with the help of wiki. When I run make, I get this error
`make -C Switch/ext_snappy/
make[1]: Entering directory '/trinity-test/Trinity/Switch/ext_snappy'
ar rc libsnappy.a snappy.o snappy-sinksource.o snappy-stubs-internal.o
make[1]: Leaving directory '/vnazzaro/trinity-test/Trinity/Switch/ext_snappy'
make -C Switch/ext/FastPFor/
make[1]: Entering directory '/vnazzaro/trinity-test/Trinity/Switch/ext/FastPFor'
CMake Error: The source directory "/db/storage/Development/Projects/Trinity/Switch/ext/FastPFor" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
Makefile:830: recipe for target 'cmake_check_build_system' failed
make[1]: *** [cmake_check_build_system] Error 1
make[1]: Leaving directory '/vnazzaro/trinity-test/Trinity/Switch/ext/FastPFor'
Makefile:63: recipe for target 'switch' failed
make: *** [switch] Error 2`

**Comment by markpapadakis** [comment]
Thank you for the report.

The new build process (please see this [wiki page](https://github.com/phaistos-networks/Trinity/wiki/How-to-Build-Trinity)) now relies on git submodules, and it should work fine.
If it doesn't, please re-open this issue and I 'll look into it.

**Comment by saxocellphone** [comment]
Thanks for the quick response! It solved that particular issue. However, I'm running into another problem when I run make with the updated instructions:
`Switch/ext_snappy/snappy.h:45:10: fatal error: 'snappy-stubs-public.h' file not found`
It looks like the file isn't found in that submodule, have you encountered something similar?

EDIT: Turns out the file is located in /ext_snappy/build, I just had to copy it one level up. 

EDIT2: Another error: `ar: =/ext/FastPFor/build/libFastPFor.a: No such file or directory`, I think the problem in line 47 of the Makefile, where there is an extra '='. It worked after I removed that.
